docs/mycoind/installation.rst

```diff
@@ -153,3 +153,7 @@ The only thing left is to start this blockchain running.
 After a few seconds this should start seeing "Commit Synced" messages.
 That means the blockchain is working away and producing new blocks,
 one a second.
+
```

Comment:
Yeah.... good points. I get used to greping for processes and killing them...

Are there any tendermint docs about that we can link to?
If not, it would warant a small section of how to cleanup and try again.